src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementCheckbox.css

```diff
@@ -0,0 +1,24 @@
+.permissionChoice {
+  display: flex;
+  align-items: center;
+  padding: 10px 0;
+  border-bottom: 1px solid rgba(54, 54, 54, 0.15);
+  color: var(--temp-grey-blue-7);
+}
+
+.permissionChoiceDescription {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding-left: 15px;
+  font-size: var(--size-tiny);
+  line-height: 18px;
+}
+
+.permissionChoiceDescription span {
+  font-size: 9px;
```

Comment:
Please don't override the label's size in this instace. I know the spec said 9px, but that's just too small.

Remove this please and leave them at their original size. _(but make sure they still align properly)_